src/commands/publish.ts

```diff
@@ -38,22 +37,21 @@ const publish = async (
 
     createBlockRequest({ displayName, publishedName }, blockData, category)
         .then((res: AxiosResponse) => {
-            const sortedVersions = sortVersions(res.data.versions);
-            const currentVersion = sortedVersions[0].version;
+            const activeVersion = 1;
 
             updateBlockSettingsFile({
+                activeVersion,
                 category,
-                currentVersion,
                 displayName,
                 id: res.data.id,
-                isActive: true,
-                isPublic: false,
-                isStaging: true,
+                versions: res.data.versions,
```

Comment:
Why are we storing all versions in the block file?

Reply:
Well, I wanted to keep it locally to later use that data to help them list which versions are being used aka major versions and their minor versions to if they wanted to rollback (later functionality) It's so we don't need to grab the versions remotely each time we need to look up. But if you rather I remove the local store I can.

Reply:
Let's remove it until we need it.

Reply:
Sounds good